src/App.jsx

```diff
@@ -14,70 +15,77 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [good, setGood] = useState('Jam');
+  const [selected, setSelected] = useState(true);
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
-
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {selected ? (
+          <>
+            {`${good} is selected`}
             <button
-              data-cy="AddButton"
+              data-cy="ClearButton"
+              onClick={() => {
+                setGood('');
+                setSelected(false);
+              }}
```

Comment:
Pls move the function inside `onClick` handler to a separate const and use as a callback here